include/web_video_server/image_streamer.h

```diff
@@ -1,7 +1,9 @@
 #ifndef IMAGE_STREAMER_H_
 #define IMAGE_STREAMER_H_
 
-#include <rclcpp/rclcpp.hpp>
+#include "async_web_server_cpp/http_request.hpp"
+#include "async_web_server_cpp/http_server.hpp"
+#include "web_video_server/utils.h"
```

Comment:
I don't see where `http_server` is being used in this header. Am I missing it?
Does `utils.h` give us the `rmw_qos_profile_t` or is it being included for some other reason?
```suggestion
#include "web_video_server/utils.h"
#include <string>
#include <optional>
```

Reply:
`string` and `optional` still need to  be included